tuber/tests/walk.py

```diff
@@ -0,0 +1,58 @@
+import os
+import tempfile
```

Comment:
I'm not familiar with the use of `volatile` over `tempfile` for creating temporary directories, I can switch too `volatile` if you prefer.